docs/modules/sidebar.md

```diff
@@ -25,7 +25,7 @@ Create and register a new sidebar.
     - {nbt}`string` **objective**: An objective used for the sidebar.
     - {nbt}`string` **display_name**: Displayed objective name. Must be a valid JSON text component.
     - {nbt}`list` **contents**: Lines displayed on the sidebar.
-      - {nbt}`list` {nbt}`string`: Must either be a valid JSON text component or an array with two JSON text components for the left and right sides.
+      - {nbt}`list` {nbt}`string`: Must either be a valid text component or an array with two text components for the left and right sides.
```

Comment:
I used "SNBT text component" in the other PR, I think we should use this here too.